packages/turndown/src/commonmark-rules.js

```diff
@@ -185,34 +190,44 @@ rules.listItem = {
         .replace(/\n+$/, '\n') // replace trailing newlines with just a single one
 
     var prefix = options.bulletListMarker + ' '
-    content = content.replace(/\n/gm, '\n    ') // indent
+    if (node.isCode===false) {
```

Comment:
```suggestion
    if (node.isCode === false) {
```

**Non-blocking/style**: This is a very minor stylistic suggestion. Feel free to ignore it :).

Reply:
fixed